src/parser/jobs/sam/modules/Kaiten.tsx

```diff
@@ -109,38 +112,40 @@ export class Kaiten extends Analyser {
 	}
 
 	private onComplete() {
+		// No suggestions if after patch 6.1
+
 		this.suggestions.add(new TieredSuggestion({
 			icon: this.data.actions.HISSATSU_KAITEN.icon,
 			content: <Trans id = "sam.kaiten.suggestion.badkaiten.content">
-				Avoid using <DataLink action="HISSATSU_KAITEN"/> on any GCDs besides <DataLink action ="IAIJUTSU"/> moves and <DataLink action = "OGI_NAMIKIRI"/>. These actions are worth it because of the potency gained per kenki spent.
+					Avoid using <DataLink action="HISSATSU_KAITEN"/> on any GCDs besides <DataLink action ="IAIJUTSU"/> moves and <DataLink action = "OGI_NAMIKIRI"/>. These actions are worth it because of the potency gained per kenki spent.
 			</Trans>,
 			tiers: SEVERITY_TIERS,
 			why: <Trans id ="sam.suggestion.badkaiten.why">
-					You used Kaiten  <Plural value={this.badKaitens} one="# time" other="# times"/> on non-optimal GCDs.
+						You used Kaiten  <Plural value={this.badKaitens} one="# time" other="# times"/> on non-optimal GCDs.
 			</Trans>,
 			value: this.badKaitens,
 		}))
 
 		this.suggestions.add(new TieredSuggestion({
 			icon: this.data.actions.HISSATSU_KAITEN.icon,
 			content: <Trans id = "sam.kaiten.suggestion.missedkaiten.content">
-				Always use <DataLink action = "HISSATSU_KAITEN"/> on <DataLink action = "MIDARE_SETSUGEKKA"/>,<DataLink action = "HIGANBANA"/> and <DataLink action="OGI_NAMIKIRI"/>. The gain on these actions from kaiten is too great to miss.
+					Always use <DataLink action = "HISSATSU_KAITEN"/> on <DataLink action = "MIDARE_SETSUGEKKA"/>,<DataLink action = "HIGANBANA"/> and <DataLink action="OGI_NAMIKIRI"/>. The gain on these actions from kaiten is too great to miss.
 			</Trans>,
 			tiers: SEVERITY_TIERS,
 			why: <Trans id ="sam.suggestion.missedkaiten.why">
-					You forgot to use Kaiten  <Plural value={this.missedKaitens} one="# time" other="# times"/> on optimal GCDs.
+						You forgot to use Kaiten  <Plural value={this.missedKaitens} one="# time" other="# times"/> on optimal GCDs.
```

Comment:
Same here, prob check the indentation for the lot since even pre-PR the why text here was already overly indented.

Reply:
I suspect VSC trying to auto fix indents when I removed the patch check borked it.